### PR TITLE
Update German install-kubectl-linux.md

### PR DESCRIPTION
**Pull request by froth**
The German installation instructions for RedHat were outdated and do not currently work.

This replaced `packages.cloud.google.com` with the current `pkgs.k8s.io` urls. 
Also adds SUSE installation instructions in german language.

I found no active german localization branch so I opened this PR on the main branch. If there is a better branch please let me know.

### CODE REVIEW

File: content/de/docs/tasks/tools/install-kubectl-linux.md [hunk 1]
```diff
@@ -163,17 +163,60 @@ Falls es benötigt wird, kann es angelegt werden. Hierzu sollte es danach von je
...
-gpgcheck=1
-gpgkey=https://packages.cloud.google.com/yum/doc/yum-key.gpg https://packages.cloud.google.com/yum/doc/rpm-package-key.gpg
-EOF
-sudo yum install -y kubectl
-```
+
+1. Hinzufügen des Kubernetes `yum` Repository. Wenn eine andere Kubernetes Version als {{< param "version" >}} installiert werden soll, muss {{< param "version" >}} im unteren Block durch die gewünschte Version ersetzt werden.
+
+   ```bash
+   # This overwrites any existing configuration in /etc/yum.repos.d/kubernetes.repo
```
Comment (utkarsh-singh1) [inline content/de/docs/tasks/tools/install-kubectl-linux.md:170]: @froth, if you are localizing the docs, localize this bash comments also.

File: content/de/docs/tasks/tools/install-kubectl-linux.md [hunk 2]
```diff
@@ -163,17 +163,60 @@ Falls es benötigt wird, kann es angelegt werden. Hierzu sollte es danach von je
...
+   baseurl=https://pkgs.k8s.io/core:/stable:/{{< param "version" >}}/rpm/
+   enabled=1
+   gpgcheck=1
+   gpgkey=https://pkgs.k8s.io/core:/stable:/{{< param "version" >}}/rpm/repodata/repomd.xml.key
+   EOF
+   ```
+{{< note >}}
+Wenn ein andere minor Version von kubectl installiert werden soll muss `/etc/yum.repos.d/kubernetes.repo` angepasst werden
+bevor `yum install` ausgeführt wird. Eine genauere Beschreibung findet sich hier
+[Changing The Kubernetes Package Repository](/docs/tasks/administer-cluster/kubeadm/change-package-repository/).
```
Comment (utkarsh-singh1) [inline content/de/docs/tasks/tools/install-kubectl-linux.md:183]: localize the `Changing The Kubernetes Package Repository`

File: content/de/docs/tasks/tools/install-kubectl-linux.md [hunk 3]
```diff
@@ -163,17 +163,60 @@ Falls es benötigt wird, kann es angelegt werden. Hierzu sollte es danach von je
...
+   sudo yum install -y kubectl
+   ```
+
+{{% /tab %}}
+
+{{% tab name="SUSE-basierte Distributionen" %}}
+1. Hinzufügen des Kubernetes `zypper` Repository. Wenn eine andere Kubernetes Version als {{< param "version" >}} installiert werden soll, muss {{< param "version" >}} im unteren Block durch die gewünschte Version ersetzt werden.
+
+   ```bash
+   # This overwrites any existing configuration in /etc/zypp/repos.d/kubernetes.repo
```
Comment (utkarsh-singh1) [inline content/de/docs/tasks/tools/install-kubectl-linux.md:198]: localize the bash comment.

File: content/de/docs/tasks/tools/install-kubectl-linux.md [hunk 4]
```diff
@@ -163,17 +163,60 @@ Falls es benötigt wird, kann es angelegt werden. Hierzu sollte es danach von je
...
+   enabled=1
+   gpgcheck=1
+   gpgkey=https://pkgs.k8s.io/core:/stable:/{{< param "version" >}}/rpm/repodata/repomd.xml.key
+   EOF
+   ```
+
+{{< note >}}
+Wenn ein andere minor Version von kubectl installiert werden soll muss `/etc/zypp/repos.d/kubernetes.repo` angepasst werden
+bevor `zypper update` ausgeführt wird. Eine genauere Beschreibung findet sich hier
+[Changing The Kubernetes Package Repository](/docs/tasks/administer-cluster/kubeadm/change-package-repository/).
```
Comment (utkarsh-singh1) [inline content/de/docs/tasks/tools/install-kubectl-linux.md:212]: localize the `Changing The Kubernetes Package Repository` sentence.

File: content/de/docs/tasks/tools/install-kubectl-linux.md (hunk 2 above)
Comment (froth) [reply content/de/docs/tasks/tools/install-kubectl-linux.md:183]: The problem is, that the linked Page has not yet been localized, I can localize the link text anyways, if that's preferred

Comment (utkarsh-singh1) [reply content/de/docs/tasks/tools/install-kubectl-linux.md:183]: It would be great if you can localize the page content.

Comment (utkarsh-singh1) [reply content/de/docs/tasks/tools/install-kubectl-linux.md:183]: Screenshot from 2024-06-27 16-16-30

But this sentence `Changing The Kubernetes Package Repository` does not seem to be fit there.

Comment (froth) [reply content/de/docs/tasks/tools/install-kubectl-linux.md:183]: yeah agreed looks weird. I will translate it

File: content/de/docs/tasks/tools/install-kubectl-linux.md [hunk 5]
```diff
@@ -163,17 +163,60 @@ Falls es benötigt wird, kann es angelegt werden. Hierzu sollte es danach von je
...
+   name=Kubernetes
+   baseurl=https://pkgs.k8s.io/core:/stable:/{{< param "version" >}}/rpm/
+   enabled=1
+   gpgcheck=1
+   gpgkey=https://pkgs.k8s.io/core:/stable:/{{< param "version" >}}/rpm/repodata/repomd.xml.key
+   EOF
+   ```
+
+{{< note >}}
+Wenn ein andere minor Version von kubectl installiert werden soll muss `/etc/zypp/repos.d/kubernetes.repo` angepasst werden
```
Comment (bene2k1) [inline content/de/docs/tasks/tools/install-kubectl-linux.md:210]:
```suggestion
Wenn eine andere minor Version von kubectl installiert werden soll muss `/etc/zypp/repos.d/kubernetes.repo` angepasst werden
```

File: content/de/docs/tasks/tools/install-kubectl-linux.md [hunk 6]
```diff
@@ -163,17 +163,60 @@ Falls es benötigt wird, kann es angelegt werden. Hierzu sollte es danach von je
...
+   [kubernetes]
+   name=Kubernetes
+   baseurl=https://pkgs.k8s.io/core:/stable:/{{< param "version" >}}/rpm/
+   enabled=1
+   gpgcheck=1
+   gpgkey=https://pkgs.k8s.io/core:/stable:/{{< param "version" >}}/rpm/repodata/repomd.xml.key
+   EOF
+   ```
+{{< note >}}
+Wenn ein andere minor Version von kubectl installiert werden soll muss `/etc/yum.repos.d/kubernetes.repo` angepasst werden
```
Comment (bene2k1) [inline content/de/docs/tasks/tools/install-kubectl-linux.md:181]:
```suggestion
Wenn eine andere minor Version von kubectl installiert werden soll muss `/etc/yum.repos.d/kubernetes.repo` angepasst werden
```

**Pull request merged**